frontend/src/modules/Survey/Components/Survey.tsx

```diff
@@ -10,8 +11,20 @@ import {
   SurveyData,
 } from 'Survey/Components/FuncsAndConsts/SurveyFunctions'
 import { Question } from '@core/Types'
+import { useEffect } from 'react'
+import { HOME_PATH } from '@core/Constants'
 
 export const Survey = () => {
+  const { search } = useLocation()
+  const history = useHistory()
+
+  const query = new URLSearchParams(search)
+  const surveyId = query.get('id')
+
+  useEffect(() => {
+    if (!surveyId) history.push(HOME_PATH)
+  }, [])
```

Comment:
Consideration for future: maybe we should give an error page or something, since redirecting students who went to `/survey` without the id query param probably did it on accident & sending them to prof page would be confusing

Reply:
Also figured out how to highlight multiple lines in a comment lol